.goreleaser.yml

```diff
@@ -25,6 +25,7 @@ builds:
       - "{{.Env.GOLDFLAGS_VERSION}}"
     goos:
       - linux
+      - darwin
```

Comment:
Can we confirm just adding this here works?

Reply:
It worked locally for me on Linux with the same version of goreleaser we use in CI (0.141.0).

Reply:
```
# ls dist/
SHA256SUMS-1.0.0-dev.txt    oasis-core-ledger_darwin_amd64
config.yaml                 oasis-core-ledger_linux_amd64
ledger-signer_darwin_amd64  oasis_core_ledger_1.0.0-dev_darwin_amd64.tar.gz
ledger-signer_linux_amd64   oasis_core_ledger_1.0.0-dev_linux_amd64.tar.gz
```

Reply:
^ this was using the `oasisprotocol/oasis-core-dev:master` docker image

Reply:
Cool. I would also like to test the GitHub Actions workflow via my private fork.

Reply:
Yeah I meant the GitHub Actions workflow :)